specifyweb/frontend/js_src/lib/components/WbPlanView/Mapper.tsx

```diff
@@ -650,6 +663,7 @@ export function Mapper(props: {
                       line: line + 1,
                     });
                 }}
+                onRemoveLine={handleDeleteLine.bind(undefined, line)}
```

Comment:
(optional) try to put smaller callbacks before longer callbacks. Otherwise, it's easy to loose this single line below the giant function that proceeds it.

Generally, callbacks are larger than other props, thus I also try to put callbacks last (consistent and more readable) - I believe we even have an ESLint rule for that